server/config.js

```diff
@@ -1,6 +1,17 @@
 import { join } from 'path'
 import { existsSync } from 'fs'
 
+require('babel-register')({
```

Comment:
I agree that this is a bit complicated because of the config, although I do like the idea that we can give the user the option to return a `Promise` instead of a function so they can do asynchronous stuff. I'll add this.